prow/plugins.yaml

```diff
@@ -128,6 +129,9 @@ lgtm:
   - kubernetes/test-infra
   review_acts_as_lgtm: true
   store_tree_hash: true
+- repos:
+  - kubernetes-sigs/kind
+  review_acts_as_lgtm: true
```

Comment:
@cjwagner I made a new entry in the list since the `store_tree_hash` seems to just be noise currently (?)

Reply:
It should work, so long as the only thing you do is squash your commits (rebasing will invalidate it)